presto-docs/src/main/sphinx/include/PrestoThriftService.thrift

```diff
@@ -200,7 +200,16 @@ struct PrestoThriftSplitBatch {
 }
 
 struct PrestoThriftSplit {
+  /**
+   * Encodes all the information needed to identify a batch of rows to return to Presto.
+   * For basic scan, includes schema name, table name and output constraint.
+   * For index scan, includes schema name, table name, set of keys to lookup and output constraint.
+   */
   1: PrestoThriftId splitId;
+  /**
+   * Identifies a subset of hosts which have access to the rows. If empty, then any host is expected
```

Comment:
The word "access" is often used in the context of authentication / authorization. I think it will read a bit better if we say "available" instead. Also, "subset" is misleading, as the list of hosts can be separate from the set of hosts used for metadata.

What do you think about this:

```
* Identifies the set of hosts on which the rows are available. If empty, then the rows
  are expected to be available on any host. The hosts in this list may be independent
  from the hosts used to serve metadata requests.

Reply:
@electrum I like this. Thanks.